packages/hooks/src/useP2PAdvertInfo.ts

```diff
@@ -9,7 +9,7 @@ const useP2PAdvertInfo = (id: string) => {
     const { subscribe, data, unsubscribe, ...rest } = useSubscription('p2p_advert_info');
 
     React.useEffect(() => {
-        subscribe({ payload: { id } });
+        if (id) subscribe({ payload: { id } });
```

Comment:
without this check, it would call the endpoint without id on mounting the advertiser page if user just navigates to it. need this so we don't call the endpoint